kodadot/nft-gallery
# Expect nothing, appreciate everything

- 🔧 bsx seo preview & canonical links
- 🔧 bsx seo preview price symbol
- [skip ci] updated code diagram
- 🧹 seo ammount decimals
- feat: remove download button + remove left/right/bottom border
- feat: add download icon
- add boolean prop and get store in `Sharing.vue`
- remove `download` action
- feat: support bsx with action button only top-border
- feat: support unique with action button only top-border
- :snake: Searching in bsx wont teleport you to rmrk
- feat: enable download only if he is owner
- feat: enable download only if he is owner
- fix bsx gallery item seo preview
- pnpm locksssssssssssssssssssssssé
- quick_fix
- quick_fix2
- accidental_pnpm_degrade
- slightly_smarter_solution
- hiding_passion_feed_everywhere
- feat: reuse `only-border-top`
- add to faq
- actual_solution
- Update utils/format/balance.ts
- Update components/bsx/Gallery/Item/GalleryItem.vue
- refactor: a faster Kodadot without passion
- delete unused code
- reverting_gallery_in_favour_of_3406
- :zap: added history table for moonsama
- :bug: query failed on empty metadata
- :bug: do not query empty metadata string
- softsama-or-moontech
- :bug: removed boolean filter from gallery
- :alien: show only NFTs with metadata
- right syntax for codeowners
- 🔧 hotfix bsx collection search
- #3435 Removing warning errors on landing page reload
- #3430 curated list doesnt change with chain/prefix change
- #3438 FIXED Focusing on search bar and then clicking outside anywhere redirects to gallery
- removing console
- history of gallery item for muti chain
- 🔧 hotfix bsx collection sort & own
- 🔧 sort collection variable visibility
- Update CollectionItem.vue
- 🔧 sort in bsx gallery & buy now
- 🔧 sorting collection item
- Update components/rmrk/Gallery/Search/SearchBar.vue
- fix: wrap element in div to prevent transition group error
- we didn't make etheria 😅
- fix conflict
- fix mistake actions
- remove useless import
- remove `download` action in Shopping action
- fix: incorrect history item
- fix: history for moonsame
- add `only-border-top` class to shareNetwork
- back to right expiration show
- fix: search history on bsx/moonsama
- feat: my offer component
- chore: rename my offers
- chore: tip for unlogin user
- feat: add 14days expiration to call
- fix: Wrap long description on gallery item
- feat: make expiration friendly
- remove debug console
- add basilisk bug issue template
- name for issue template must be unique
- mmm labels
- issue template basilisk bug form add bug label
- remove seconds
- add months
- 🔧 sorting on rmrk & bsx
- 🔧 sorting
- 🔧 rmrk sorting undefined
- 🔧 rmrk sorting undefined
- feat: add collection column and swap order
- feat: Expand ability to set offer expiration in days
- remove useless import
- fix typo
- Update AvailableActions.vue
- fix advice
- Update AvailableActions.vue
- replace longer part to `...`
- fix: request error from sort by condition
- fix: Spotlight doesnot work
- Show user balance at minting screen
- feat: replace range slider with input fields
- refactor: reuse MyBalance component
- delete unused code
- remove useless import
- reuse accept offer function
- add myOffer to nav
- feat: expiration column for myOffers
- Update SearchBar.vue
- #3203 add table to show currently standing offers for user
- removing unused props
- validation
- value check
- fix: reload page after burning
- placeholders for bug reporting bsx
- one by one, placeholder will be forever in bug form
- name better basilisk
- update
- [skip ci] updated code diagram
- fix: Duplicate of items in collection
- rename AccountBalance
- fix: show only acceptable offer
- fix: url route error
- fixes
- add collection offer
- remove useless import
- using offer query, name shorten
- making columns sortable
- fix: incorrect number of NFTs in collection
- fix: ts
- #3433 Search bar suggestions changes
- feat: enable `buy now` when click `apply`
- bugfix: `Gallery.vue` fetch params error
- bugfix: ksm decimals
- 🔧 collections cards hover
- 🔧 collections cards hover
- 🔧 collections cards hover
- fix: rename incomingoffers path name
- feat: history component for item
- fix: add guard and remove concat data
- fix: url route error on stats
- fix: collection item url
- rename incoming offers
- bugfix: disableApply + validation message position change
- refactor: change `balance` to `amount`
- feat: add filter status
- remove debug console
- sort added in status
- call redirect function only in case of enter pressed
- #3501 Pagination for global offers for Basilisk
- #3492 Small UI change
- minor change
- fix: Wrong text after accepting incoming offer
- bugfix: >= and step
- feat: sort collection / name / date
- 🔧 fix price gt 0
- feat: update sort i18
- disable strict peer deps (#41)
- feat: title for offer table
- ref: remove unnecessary template
- pin vue version
- fix: sorting by price on offer table does not work correctly
- remove formatBalance
- grammar_fix
- fix: search gallery on bsx
- fix: Sorting by price should always keep Buy Now on
- fix: reset collections sort i18
- Update components/bsx/Offer/OfferTable.vue
- changes
- #3508 Sorting by name collection fix
- remove useless class
- fix: disable minting button if not in range
- fix: correctly handle input and conversion
- chore: keep consistency in code
- [skip netlify] Update Bumped minor version of packages
- Update components/bsx/Offer/OfferTable.vue
- fix: border
- fix: collection error
- disable prefer-symlinked-executables
- fix: incorrect text on above offers table
- fix: cant mint if I dont want to put NFT on sale
- cleanup merge conlfict issue
- margin change
- fix: some 3D nft in /collection/{id} could not display correctly
- fix: default select on bsx
- fix duplicate attribute and remove useless comment
- add `displayCollection` to control column collection display
- [skip netlify] Update Bumped minor version of packages
- change OffersUserTable file path
- fix: price sort
- Add ability to sort by time in incoming offers
- fix: use urlPrefix instead
- extend offerMixins
- feat: hidden `westend` `westmint` in production env
- #3563 Add sorting in users offer table
- fix: The asset instance ID has already been used for an asset
- Update components/bsx/Offer/OffersUserTable.vue
- error fix

**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type


### What's new?

- [ ] Requires deployment <rubick/magick_issue>

### Before submitting Pull Request, please make sure:


### Optional


### Had issue bounty label?


### Community participation


### Screenshot
